src/types.ts

```diff
@@ -179,9 +179,10 @@ export interface EuHealthCertDocument {
   t: EuTestParams[];
 }
 export interface EuHealthCert extends EuHealthCertDocument {
-  meta: NotarisationMetadata;
+  meta: notarise.NotarisationMetadata;
 } 
 
 export interface EuHealthCertQr {
-  qrData?: string;
+  type: string;
+  qrData: string;
 }
```

Comment:
These types shouldn't be needed anymore right?

Instead, it can be defined like this: https://github.com/Notarise-gov-sg/healthcert-renderer/blob/master/src/templates/healthcert/types.tsx#L112